app/src/main/java/org/permanent/permanent/network/NetworkClient.kt

```diff
@@ -634,6 +634,12 @@ class NetworkClient(private var okHttpClient: OkHttpClient?, context: Context) {
         return tagService.unlinkTags(requestBody)
     }
 
+    fun deleteTags(tags: List<Tag>): Call<ResponseVO> {
+        val request = toJson(RequestContainer().addTagIds(tags))
+        val requestBody: RequestBody = request.toRequestBody(jsonMediaType)
+        return tagService.unlinkTags(requestBody)
```

Comment:
I believe you forgot to change the function here.

Reply:
the method name is somewhat unfortunate, but the endpoint is the correct one.

@POST("tag/delete")
fun unlinkTags(@Body requestBody: RequestBody): Call<ResponseVO>